Quarkus cli: `quarkus run` of picocli app with Gradle Kotlin DSL fails

### Describe the bug

I tried to run a quarkus cli app with argument: '-c -w --val 1' but it fails. The app was created with kotlin and Gradle Kotlin DSL.
Even though the  `quarkusRun` gradle tasks exists, I get this error with a `quarkus run -Dquarkus.args='-c -w --val 1'`: `[ERROR]  Unable to build: Unknown action for GRADLE_KOTLIN_DSL`

### Expected behavior

Similar behavior as `./gradlew quarkusRun -Dquarkus.args='-c -w --val 1'` 

### Actual behavior

`[ERROR]  Unable to build: Unknown action for GRADLE_KOTLIN_DSL`

### How to Reproduce?

- Create a Quarkus CLI app (with Picocli) that uses Kotlin and Gradle with Kotlin DSL `quarkus create cli --name="quarkus-cli-demo" --kotlin --gradle-kotlin-dsl --wrapper`
- Open the folder `co code-with-quarkus`
- Run the app `quarkus run -Dquarkus.args='-c -w --val 1'`

-> Event though the  `quarkusRun` gradle tasks exists, the above command fails with this error `[ERROR]  Unable to build: Unknown action for GRADLE_KOTLIN_DSL`

### Output of `uname -a` or `ver`

NA (I'm on windows)

### Output of `java -version`

openjdk version "21.0.2" 2024-01-16 LTS

### Quarkus version or git rev

3.6.7

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Gradle 8.5

### Additional information

_No response_

--- a/devtools/cli/src/main/java/io/quarkus/cli/Deploy.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/Deploy.java
@@ -16,7 +16,7 @@ import picocli.CommandLine;
 public class Deploy extends BuildToolDelegatingCommand {
 
     private static final Map<BuildTool, String> ACTION_MAPPING = Map.of(BuildTool.MAVEN, "quarkus:deploy",
-            BuildTool.GRADLE, "deploy");
+            BuildTool.GRADLE, "deploy", BuildTool.GRADLE_KOTLIN_DSL, "deploy");
 
     @Override
     public Map<BuildTool, String> getActionMapping() {

--- a/devtools/cli/src/main/java/io/quarkus/cli/Run.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/Run.java
@@ -9,7 +9,7 @@ import picocli.CommandLine;
 public class Run extends BuildToolDelegatingCommand {
 
     private static final Map<BuildTool, String> ACTION_MAPPING = Map.of(BuildTool.MAVEN, "quarkus:run",
-            BuildTool.GRADLE, "quarkusRun");
+            BuildTool.GRADLE, "quarkusRun", BuildTool.GRADLE_KOTLIN_DSL, "quarkusRun");
 
     @CommandLine.Option(names = { "--target" }, description = "Run target.")
     String target;

--- a/devtools/cli/src/main/java/io/quarkus/cli/image/Build.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/image/Build.java
@@ -20,7 +20,7 @@ public class Build extends BaseImageCommand {
     private static final String FALLBACK_BUILDER = "docker";
 
     private static final Map<BuildTool, String> ACTION_MAPPING = Map.of(BuildTool.MAVEN, "quarkus:image-build",
-            BuildTool.GRADLE, "imageBuild");
+            BuildTool.GRADLE, "imageBuild", BuildTool.GRADLE_KOTLIN_DSL, "imageBuild");
 
     @Override
     public void populateContext(BuildToolContext context) {

--- a/devtools/cli/src/main/java/io/quarkus/cli/image/Push.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/image/Push.java
@@ -13,7 +13,7 @@ public class Push extends BaseImageCommand {
 
     protected static final String QUARKUS_CONTAINER_IMAGE_EXTENSION = "io.quarkus:quarkus-container-image";
     private static final Map<BuildTool, String> ACTION_MAPPING = Map.of(BuildTool.MAVEN, "quarkus:image-push",
-            BuildTool.GRADLE, "imagePush");
+            BuildTool.GRADLE, "imagePush", BuildTool.GRADLE_KOTLIN_DSL, "imagePush");
 
     private static final String QUARKUS_CONTAINER_IMAGE_BUILD = "quarkus.container-image.build";
     private static final String QUARKUS_CONTAINER_IMAGE_USERNAME = "quarkus.container-image.username";

